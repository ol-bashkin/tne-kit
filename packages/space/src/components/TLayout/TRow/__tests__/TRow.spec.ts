import { mount } from "@vue/test-utils";
import TRow from "../TRow";

describe("WRow.ts", () => {
  test("should have an t-row class", () => {
    const wrapper = mount(TRow, {
      propsData: {
        row: "2",
      },
    });
    expect(wrapper.classes()).toContain("t-row");
    expect(wrapper.html()).toMatchSnapshot();
  });
});
