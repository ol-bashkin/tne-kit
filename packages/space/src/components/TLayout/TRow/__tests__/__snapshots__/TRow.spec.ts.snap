// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WRow.ts should have an t-row class 1`] = `<div class="t-row row row-2"></div>`;
